### ISSUE TITLE
The interactive device verification with user @drew:drewdevault.com got canceled: Error: No devices could be verified.

### ISSUE THREAD

**Issue by ddevault** [poljar/weechat-matrix#324]
While attempting to verify weechat-matrix

**Comment by ddevault** [comment]
Bah. Thanks!

**Comment by ddevault** [comment]
How does one do this? I was looking for something like this in element (web & android) but was met without success.

**Comment by poljar** [comment]
You go to the `RoomInfo` (right corner) -> `People` -> Your own user tab. This gives you a list of your devices, clicking on a device gives you a slightly different dialog:

**Comment by ddevault** [comment]
I have not yet joined any rooms, just trying to work with direct messages for now. I tried to follow your instructions, but the room info pop-up does not have a people tab and I cannot find verification stuff anywhere else.

I do have this in my settings:

![](https://l.sr.ht/bx6R.png)

But hitting "verify" causes no change in behavior in weechat-matrix. I tried to manually use `/olm verification accept` without waiting for a prompt to do so, but this gave a "no active key verification found for device" message. If I cancel the verification in element, it shows the same error message I opened this issue with, regardless of whether or not I proactively ran `/olm verification accept` during the process.

In any case, it seems like all of this is very unreliable and unfinished, and I don't have the patience to troubleshoot it further. I will try again in a year or two and see how it's come along since.
